gc/policy/policy.go

```diff
@@ -0,0 +1,217 @@
+package policy
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/containerd/containerd/log"
+	"github.com/containerd/containerd/metadata"
+	"github.com/containerd/containerd/plugin"
+)
+
+type config struct {
+	PauseThreshold float64 `toml:"pause_threshold"`
+
+	ScheduleOnDelete  bool `toml:"on_delete"`
+	MutationThreshold int  `toml:"mutation_threshold"`
+
+	ScheduleDelayMs int `toml:"schedule_delay_ms"`
+	StartupDelayMs  int `toml:"startup_delay_ms"`
+}
+
+func init() {
+	plugin.Register(&plugin.Registration{
+		Type: plugin.GCPlugin,
+		ID:   "policy",
```

Comment:
policy -> defaultpolicy?

Reply:
no, keep it simple.  calling something default<whatever> just makes things ugly and its not a default, it can be configured.

if anything, the name could be more specific as to what type of gc policy it is.  its a scheduled or timed based gc policy, something to that effect